src/main/java/com/ibm/watson/modelmesh/ModelMesh.java

```diff
@@ -916,12 +918,33 @@ private static Metrics setUpMetrics() throws Exception {
                 params.put(kv[0], kv[1]);
             }
         }
+        String infoMetricConfig = getStringParameter(MMESH_CUSTOM_ENV_VAR, null);
+        Map<String, String> infoMetricParams;
+        if (infoMetricConfig == null) {
+            logger.info("{} returned null", MMESH_CUSTOM_ENV_VAR);
+            infoMetricParams = Collections.emptyMap();
```

Comment:
It's probably better to use `null` to indicate disabled rather than an empty map.